app/assets/stylesheets/style.scss

```diff
@@ -328,4 +350,46 @@ label:hover {
   }
 }
 
+#footer {
+  position: fixed;
```

Comment:
Properties should be ordered background-color, border-top, bottom, height, margin-top, position, width